terraform/files/bin/delete_cluster.sh

```diff
@@ -40,6 +37,9 @@ kubectl $KCONTEXT get cronjobs --all-namespaces -o=jsonpath="{'set -x\n'}{range
 # Delete all Jobs
 echo "Deleting all Jobs ..."
 kubectl $KCONTEXT get jobs --all-namespaces -o=jsonpath="{'set -x\n'}{range .items[*]}{'kubectl $KCONTEXT delete job '}{.metadata.name}{' -n '}{.metadata.namespace}{' --grace-period=0 --force\n'}{end}" | bash
+# Delete pods with persistent volume claims
+echo "Deleting pods with persistent volume claims ..."
+kubectl $KCONTEXT get pods --all-namespaces -o=jsonpath="{'set -x\n'}{range .items[?(@.spec.volumes[*].persistentVolumeClaim.claimName)]}{'kubectl $KCONTEXT delete pod '}{.metadata.name}{' -n '}{.metadata.namespace}{' --grace-period=0 --ignore-not-found=true\n'}{end}" | bash
```

Comment:
And what about `pods` on line 51? Won't they also be recreated by existing resources below(deployment,daemonset,statefulset)?

Reply:
Most likely. Moved these lines as well :)